lib/experimental/Navigation/Carousel/index.tsx

```diff
@@ -0,0 +1,85 @@
+import { cn } from "@/lib/utils"
+import {
+  CarouselContent,
+  CarouselDots,
+  CarouselItem,
+  CarouselNext,
+  CarouselPrevious,
+  Carousel as ShadCarousel,
+} from "@/ui/carousel"
+import Autoplay from "embla-carousel-autoplay"
+import { WheelGesturesPlugin } from "embla-carousel-wheel-gestures"
+import React from "react"
+
+interface CarouselProps {
+  children: React.ReactNode
+  showArrows?: boolean
+  showDots?: boolean
+  autoplay?: boolean
+  delay?: number
+  itemClassName?: string
+}
+
+export const Carousel = ({
+  children,
+  showArrows = true,
+  showDots = true,
+  autoplay = false,
+  delay = 3000,
+  itemClassName = "sm:basis-1/2 md:basis-1/3 lg:basis-1/4",
```

Comment:
I believe we shouldn't have a default size for the items, we should always be explicit when using this component with the amount of columns/items per slide we want

Reply:
Left at 1 item by default, and other breakpoints can be defined later when using the component.